crates/common/tedge_config/Cargo.toml

```diff
@@ -6,16 +6,19 @@ edition = "2021"
 rust-version = "1.58.1"
 
 [dependencies]
+thiserror = "1.0"
+strum_macros = { version = "0.24", optional = true }
+tracing = { version = "0.1", features = [] }
+
 certificate = { path = "../certificate" }
 serde = { version = "1.0", features = ["derive"] }
 tedge_utils = { path = "../tedge_utils" }
 tempfile = "3.2"
-thiserror = "1.0"
 toml = "0.5"
 url = "2.2"
 
 [dev-dependencies]
+anyhow="1.0"
```

Comment:
Does cargo have a way at sorting the dependencies to avoid re-ordering in each PR? It makes it a difficult to see what changed.

Reply:
Definitely for another PR. But that doesn't prevent you to try to keep dependencies ordered meanwhile ;-)

Reply:
I already did for this ;)